### PR TITLE
Frontend join page

### PR DESCRIPTION
**Pull request by jcderry5**
This branch add a form tag to the join room html file and minor styling to the join room HTML page

### CODE REVIEW

**Comment by jcderry5** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fcapstone-step150-2020%2Fpull%2F1) for more info**.

File: src/main/webapp/join-room.html [hunk 1]
```diff
@@ -3,6 +3,13 @@
...
+    <style>
+      body{
+          margin: auto;
+          text-align: center;
+      }
+    </style>
+
   </head>
   <body>
     <div id="join-room-box">
```
Comment (tianxiao25) [inline src/main/webapp/join-room.html:15]: Delete the whole target folder. That's binary files. Shouldn't be included!

File: target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Tue Jun 23 18:58:32 UTC 2020
+#Wed Jun 24 15:41:45 UTC 2020
```
Comment (tianxiao25) [inline target/maven-archiver/pom.properties:2]: Same here. Delete the target folder.

File: src/main/webapp/join-room.html (hunk 1 above)
Comment (jcderry5) [reply src/main/webapp/join-room.html:15]: The target directory cannot be deleted and still run on localhost. I put in a .gitignore to stop adding target to all future commits, though.

**Comment by jcderry5** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fcapstone-step150-2020%2Fpull%2F1) for more info**.

@googlebot I signed it!

File: src/main/webapp/join-room.html [hunk 3]
```diff
@@ -3,6 +3,13 @@
   <head>
```
Comment (tianxiao25) [inline src/main/webapp/join-room.html:3]: add a 'page' folder under webapp and move this to there.

**Pull request closed without merging**